pytest_fixtures/component/http_proxy.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+
+
+def create_http_proxy(sat, org, http_proxy_type):
```

Comment:
Could this be moved under APIFactory class and made available to be used across robottelo by satellite object?

Reply:
updated the pr.